Methods dataset (#2)
greenelab/annorxiver_manuscript: content/03.methods.md

### Datasets

#### BioRxiv

> BioRxiv [@doi:10.1101/833400] is a repository that contains biological and biomedical research preprints.

Reviewer: ```suggestion
BioRxiv [@doi:10.1101/833400] is a repository of biological and biomedical research preprints.

```

> We downloaded an xml snapshot of this repository on February 3, 2020 and this snapshot contained 98,023 total preprints.

Reviewer: ```suggestion
We downloaded an xml snapshot of this repository on February 3, 2020 from _WHERE_ that contained the full text and image content of 98,023 preprints.
```

> 26,905 out of 98,023 are preprints that contain more than one version as bioRxiv allows users to upload revisions and corrections.

Reviewer: ```suggestion
Preprionts on bioRxiv are versioned, and 26,905 of 98,023 contained more than one version.
```

> We fixed this duplication issue by focusing only on the latest version of preprint. 

Reviewer: ```suggestion
When preprints had multiple versions we used only the latest one. 
```

> Preprints in this snapshot can fall into one of the twenty-nine different categories with neuroscience being the most popular (Figure {@fig:biorxiv_categories}).

Reviewer: ```suggestion
Preprints in this snapshot were grouped into one of twenty-nine different categories.
```
Text and figure should go in results if you want to start getting that together: "with neuroscience being the most popular (Figure {@fig:biorxiv_categories})."

> Furthermore, each preprint depicts of the following content types: presenting a new result, confirming a previous finding or presenting a contradiction to a previous finding.

Reviewer: ```suggestion
Each preprint was also classified as a new result, confirmatory finding, or contradictory finding.
```

> Some preprints in this snapshot have even been withdrawn from bioRxiv.

Reviewer: ```suggestion
Some preprints in this snapshot have been withdrawn from bioRxiv.
When a preprint is withdrawn, X happens.
Because we use only the latest version, (what happens to these preprints).
```

> Overall, majority of preprints are writing to present new research results.

Reviewer: This line should go in the results.

![

> BioRxiv has an influx of neuroscience preprints followed by bioinformatics and then microbiology.

Reviewer: This figure should go in the results.

This bar chart depicts the number of preprints that fall into each category of bioRxiv.
](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/exploratory_data_analysis/output/figures/preprint_category.png){#fig:biorxiv_categories}

#### PubMed Central

> PubMed Central (PMC) [@doi:10.1073/pnas.98.2.381] is a repository that contains open access to published research articles.

Reviewer: ```suggestion
PubMed Central (PMC) [@doi:10.1073/pnas.98.2.381] is a repository that contains free-to-read articles.
PMC contains two types of contributions: closed access articles from research funded by the United States National Institutes of Health and articles published under Gold Open Access _REFERENCE FOR THIS_ publishing schemes.
HOW DO CLOSED ACCESS ARTICLES GET HERE AND HOW MANY ARE THERE.
HOW DO OPEN ACCESS ARTICLES GET HERE AND HOW MANY ARE THERE.
WHICH SET IS AVAILABLE FOR LITERATURE MINING?
```

> We downloaded a snapshot of this repository on January 31, 2020.

Reviewer: ```suggestion
We downloaded a snapshot of the open access subset on January 31, 2020.
```

> This snapshot contains 9,440 unique journals with 2,719,226 different articles (Figure {@fig:pmc_statistics}).

Reviewer: ```suggestion
This snapshot contains 2,719,226 different articles from 9,440 unique journals (Figure {@fig:pmc_statistics}).
```
Would suggest a light rephrase and this should also go in results if there's going to be a figure.

> We subsampled this snapshot to only use research articles, which leaves 1,977,651 unique articles for our analyses.

Reviewer: Would recommend discussing the types of articles that exist here (as with bioRxiv) and then say you selected only research articles. Then move the precise numbers to the results section so that you can have the figure.

![

> Majority of PubMed Central's (PMC) articles are research articles.

Reviewer: Results

### Datasets

#### BioRxiv
BioRxiv [@doi:10.1101/833400] is a repository of biological and biomedical research preprints.
We downloaded an xml snapshot of this repository on February 3, 2020 from   bioRxiv's Amazon S3 resource [@https://www.biorxiv.org/tdm] that contained the full text and image content of 98,023 preprints.
Preprints on bioRxiv are versioned, and 26,905 of 98,023 contained more than one version.
When preprints had multiple versions, we used only the latest one. 
Preprints in this snapshot were grouped into one of twenty-nine different categories.
Each preprint was also classified as a new result, confirmatory finding, or contradictory finding.
Some preprints in this snapshot have been withdrawn from bioRxiv.
When a preprint is withdrawn, its content is replaced with the reason for withdrawal.

> Because we use only the latest version, withdrawn preprints only contain withdrawn statements within their content.

Reviewer: ```suggestion
Because we used the latest version, withdrawn preprints in our analysis contained only statements indicating their removal.
```

#### PubMed Central
PubMed Central (PMC) [@doi:10.1073/pnas.98.2.381] is a repository that contains free-to-read articles.

> PMC contains two types of contributions: closed access articles from research funded by the United States National Institutes of Health (NIH) and articles published under Gold Open Access [@doi:10.1007/s12471-017-1064-2] publishing schemes.

Reviewer: ```suggestion
PMC contains two types of contributions: closed access articles from research funded by the United States National Institutes of Health (NIH) appearing after an embargo period and articles published under Gold Open Access [@doi:10.1007/s12471-017-1064-2] publishing schemes.
```

Paper availability within PMC is largely dependent on the journal's participation level [@https://www.ncbi.nlm.nih.gov/pmc/about/submission-methods/].

> Individual journals have can fall into the following categories: fully participation in submitting articles to PMC, selective participation as a select few of papers are submitted to PMC, only submit papers according to NIH's public access policy [@https://grants.nih.gov/grants/policy/nihgps/html5/section_8/8.2.2_nih_public_access_policy.htm], and lastly no participation at all.

Reviewer: ```suggestion
Individual journals have can fully participate in submitting articles to PMC, selectively participate sending only a few few of papers to PMC, only submit papers according to NIH's public access policy [@https://grants.nih.gov/grants/policy/nihgps/html5/section_8/8.2.2_nih_public_access_policy.htm], or not participate at all.
```

> As of September 2019, PMC has a total of 5,725,819 articles available [@https://www.ncbi.nlm.nih.gov/pmc/about/intro/].

Reviewer: ```suggestion
As of September 2019, PMC had 5,725,819 articles available [@https://www.ncbi.nlm.nih.gov/pmc/about/intro/].
```

> Out of these 5 million articles about 3 million are considered open access and available for text processing systems [@doi:10.1093/bioinformatics/btz070;@doi:10.1093/nar/gkz389].

Reviewer: ```suggestion
Out of these 5 million articles, about 3 million were open access and available for text processing systems [@doi:10.1093/bioinformatics/btz070;@doi:10.1093/nar/gkz389].
```

We downloaded a snapshot of this open access subset on January 31, 2020.

> This snapshot contains papers such as literature reviews, book reviews, editorials, case reports, research articles and more; however, we subsampled it to focus solely on research articles.

Reviewer: ```suggestion
This snapshot contains papers such as literature reviews, book reviews, editorials, case reports, research articles and more; however, we used only the research articles.
```
